### PR TITLE
Update callbacks samples to use .NET Core MVC

### PR DESCRIPTION
**Pull request by mikeminutillo**
The Callbacks samples were using ASP.NET MVC. This PR updates them to run in .NET Core MVC.

They are targeted at .NET Core 3.1 to match [all of the other .NET Core Web projects in docs](https://github.com/Particular/docs.particular.net/search?q=Sdk%3D%22Microsoft.NET.Sdk.Web%22). 

This also allowed updating these projects to use the new .NET SDK style project format.

### CODE REVIEW

File: samples/callbacks/Callbacks_3/WebSender/HomeController.cs [hunk 1]
```diff
@@ -17,8 +24,7 @@ public async Task<ActionResult> SendEnumMessage()
         var message = new EnumMessage();
         var sendOptions = new SendOptions();
         sendOptions.SetDestination("Samples.Callbacks.Receiver");
-        var endpointInstance = MvcApplication.EndpointInstance;
-        var statusTask = endpointInstance.Request<Status>(message, sendOptions);
+        var statusTask = messageSession.Request<Status>(message, sendOptions);
```
Comment (danielmarbach) [inline samples/callbacks/Callbacks_3/WebSender/HomeController.cs:27]: I would find 

```
var status = await messageSession.Request<Status>(message, sendOptions);
```

more user-friendly for a sample. Comment applies for all the other cases too.

File: samples/callbacks/Callbacks_3/WebSender/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+public class Startup
```
Comment (danielmarbach) [inline samples/callbacks/Callbacks_3/WebSender/Startup.cs:4]: More as a general question, feel free to ignore:

Is using the startup class still a thing or aren't most people moving towards using the `ConfigureXYZ` overloads on the builder in the program main?

**Review by danielmarbach — APPROVED** [review]

File: samples/callbacks/Callbacks_3/WebSender/Startup.cs (hunk 2 above)
Comment (mikeminutillo) [reply samples/callbacks/Callbacks_3/WebSender/Startup.cs:4]: I kept it consistent with the other samples to make it easier to update them all at once:

- https://github.com/Particular/docs.particular.net/blob/0874d4df530bd71014e8e8d79ede7036f5f772f5/samples/dependency-injection/aspnetcore/Core_7.2/Sample/Program.cs#L33
- https://github.com/Particular/docs.particular.net/blob/0874d4df530bd71014e8e8d79ede7036f5f772f5/samples/web/send-from-aspnetcore-webapi/Core_7.2/WebApplication/Program.cs#L23
- https://github.com/Particular/docs.particular.net/blob/ed8459f17ec30a56309b8a16c9fb5beaf792c363/samples/web/send-from-aspnetcore-webapi/Core_8/WebApplication/Program.cs#L23
- etc...

File: samples/callbacks/Callbacks_3/WebSender/HomeController.cs (hunk 1 above)
Comment (mikeminutillo) [reply samples/callbacks/Callbacks_3/WebSender/HomeController.cs]: Good call. I'll update these.

**Pull request merged**